infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -113,6 +111,19 @@ public void prefixTrim(@Nonnull long address) {
         }
     }
 
+    /**
+     * Reset the log unit node.
+     */
+    public void reset() {
+        try {
+            CompletableFuture<Void> cf = new CompletableFuture<>();
+            operationsQueue.add(new BatchWriterOperation(Type.RESET, null, null, null, cf));
+            cf.get();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)